src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const stateCopy = state;
+  const removeKey = function(actionObject) {
+    for (const keyToRemove of actionObject.keysToRemove) {
+      delete stateCopy[keyToRemove];
+    }
+  };
+
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
You can use destructuring here: const { type, extraData, keysToRemove } = action